source/features/stop-redirecting-in-notifications-bar.tsx

```diff
@@ -0,0 +1,25 @@
+import delegate from 'delegate-it';
+import features from '../libs/features';
+import * as pageDetect from '../libs/page-detect';
+
+function onNotificationActionClick(event: delegate.Event<MouseEvent, HTMLButtonElement>): void {
```

Comment:
I like that the way this function is written doesn't break successive non-alt clicks. Can you mention this fact in a comment?